apollo-runtime/src/main/java/com/apollographql/apollo/http/OkHttpExecutionContext.java

```diff
@@ -0,0 +1,46 @@
+package com.apollographql.apollo.http;
+
+import com.apollographql.apollo.api.ExecutionContext;
+import okhttp3.Response;
+import org.jetbrains.annotations.NotNull;
+
+import static com.apollographql.apollo.api.internal.Utils.checkNotNull;
+
+/**
+ * Http GraphQL execution context, provides access to the raw {@link okhttp3.Response} response.
+ */
+public class OkHttpExecutionContext implements ExecutionContext.Element {
```

Comment:
Just a small thing. Since all modules has Kotlin dependency transitively, it is also added to runtime module. So it is ok to write this now with Kotlin.